runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/io/DataflowBigQueryIOTest.java

```diff
@@ -40,6 +39,30 @@
  */
 public class DataflowBigQueryIOTest {
   @Test
+  public void testTableSourcePrimitiveDisplayData() {
+    DisplayDataEvaluator evaluator = DataflowDisplayDataEvaluator.create();
+    BigQueryIO.Read.Bound read = BigQueryIO.Read
+        .from("project:dataset.tableId")
+        .withoutValidation();
+
+    Set<DisplayData> displayData = evaluator.displayDataForPrimitiveTransforms(read);
+    assertThat("BigQueryIO.Read should include the table spec in its primitive display data",
+        displayData, hasItem(hasDisplayItem("table")));
```

Comment:
This assertion reads unfortunately -- `hasItem(hasDisplayItem("table"))` -- but I don't see a clean way to fix it without introducing a data structure to wrap the `Set<DisplayData>`. Let's leave it unless you have any thoughts.